app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -88,6 +88,12 @@ function ReviewIssuesStep({
     setHasDismissed(true);
   }
 
+  // let FormSteps know, via FormStepsContext, whether this page
```

Comment:
When returning back to the review issues step after opting to proof in-person, a lot of the page content is missing:

Before IPP|After "Back"
---|---
![image](https://user-images.githubusercontent.com/1779930/178284353-553fc12c-dc04-4447-a448-25c0aed86528.png)|![image](https://user-images.githubusercontent.com/1779930/178284286-fa817a13-2f37-4d2b-a236-33170f744b94.png)

I think it has to do with how `toNextStep` will clear out any errors, and those errors are how we generate the content shown in the review_issues warning.

Reply:
I wrote a bug ticket LG-6874 to fix this